src/patterns/SaleHeaderBanner/SaleHeaderBanner.test.tsx

```diff
@@ -1,7 +1,7 @@
 import { render, screen } from '@testing-library/react';
 import { describe, it, expect } from 'vitest';
 import SaleHeaderBanner, { SaleHeaderBannerProps } from './SaleHeaderBanner';
-import { AuctionStatus } from '../../types/commonTypes';
+import { LotAuctionStatus } from '../../types/commonTypes';
```

Comment:
Same comment about status. SaleHeaderBanner is concerned with the status of the auction and not lot